canvas_modules/common-canvas/src/common-properties/controls/toggletext/toggletext.scss

```diff
@@ -19,9 +19,11 @@
 	&.hide {
 		display: none;
 	}
-	button > img {
-		height: 16px;
-		padding: 0 $spacing-03 0 0;
+	.toggletext-icon {
+		position: absolute;
```

Comment:
When toggletext button is in the table, button text is overlapped by icon because of `position: absolute` style.
Should I remove this style when `tableControl` is set to true?

![toggletext-in-table](https://user-images.githubusercontent.com/25124000/107294420-34ab3580-6a22-11eb-94fc-c5f5c160d1e8.gif)

Reply:
What does it look like if you remove the styling?

Reply:
**`position: absolute` style added -**
![Screen Shot 2021-02-10 at 8 32 57 AM](https://user-images.githubusercontent.com/25124000/107540533-01c88500-6b7b-11eb-860d-f7e1f1a76c33.png)

![Screen Shot 2021-02-10 at 8 33 26 AM](https://user-images.githubusercontent.com/25124000/107540563-08ef9300-6b7b-11eb-8884-f78e9e8b6d0c.png)


**`position: absolute` style removed -**
![Screen Shot 2021-02-10 at 8 34 23 AM](https://user-images.githubusercontent.com/25124000/107540591-1147ce00-6b7b-11eb-8d34-af1f3706e3d1.png)

![Screen Shot 2021-02-10 at 8 34 36 AM](https://user-images.githubusercontent.com/25124000/107540612-1573eb80-6b7b-11eb-9de8-f26ebbe555e7.png)

Reply:
A few changes we should make.
- Add `bx--btn__icon` to the class of the images.  (this fixes a number of styling issues)
- In a table make the button a ghost button instead
- With those changes the scss for toggletext can be updated to 
```
.properties-toggletext {
	&.hide {
		display: none;
	}
	&.error {
		button {
			border-bottom: 2px solid $support-01;
		}
	}
	&.warning {
		button {
			border-bottom: 2px solid $support-03;
		}
	}
}

// toggle control in table needs to fit within column
.properties-table-cell-control {
	.properties-toggletext button {
		padding: 0 $spacing-02 0 $spacing-02;
	}
}
```
The left padding in the table makes the text look a little misaligned.  See if you can easily fix that if not it's probably ok.

Reply:
Thanks Matt. I updated the code. Didn't remove left padding in table.

Left padding in the table is added to leave some space between error/warning icon and button -
![Screen Shot 2021-02-11 at 10 32 19 AM](https://user-images.githubusercontent.com/25124000/107681881-820ee900-6c54-11eb-93a8-b648ccf7bafa.png)

If I remove the left padding, error/warning icon is shown just above the button -
![Screen Shot 2021-02-11 at 10 23 56 AM](https://user-images.githubusercontent.com/25124000/107681937-92bf5f00-6c54-11eb-8540-7923fea40b8a.png)